mappings/net/minecraft/item/AxeItem.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS atr net/minecraft/item/AxeItem
+	FIELD a STRIPPED_LOGS_MAP Ljava/util/Map;
```

Comment:
ShovelItem has a very similar map. I would call this something more generic as such, such as "BLOCK_TRANSFORMATIONS_MAP"

Reply:
Changed